src/components/Details/VmDetails/VmDetails.js

```diff
@@ -231,6 +234,9 @@ export class VmDetails extends React.Component {
 
                   <dt>Pod</dt>
                   <dd>{PodResourceLink ? <PodResourceLink /> : DASHES}</dd>
+
+                  <dt>Boot Order</dt>
```

Comment:
this could make a sense also for VmTemplateDetails

Reply:
Agreed! Done!